ftests/metrics/src/main/resources/default-test-metrics.conf

```diff
@@ -3,6 +3,11 @@ enabled = true
 reporter.enabled = false
 
 ######################################
+# List of enabled metrics item groups
```

Comment:
Let's not worry about filtering item groups for now. We should just assume that things we annotate will be reported for now, which is why we should be careful to only annotate things that we really care about.

When we put together a filtering system, it's going to need to be a little more rich than this, I think, since it's likely we're going to have metrics next to one another in the same class where we will want one enabled and the other disabled.